source/ui/dataman/AMScanViewUtilities.cpp

```diff
@@ -766,41 +432,39 @@ void AMScanViewSingleSpectrumView::onDataPositionChanged(const AMnDIndex &index)
 void AMScanViewSingleSpectrumView::onSelectedRectChanged(const AMnDIndex &start, const AMnDIndex &end)
 {
 	if (isVisible()){
-
 		addMultipleSpectra_ = true;
 		updatePlot(start, end);
 	}
 }
 
-void AMScanViewSingleSpectrumView::setAxisInfo(AMAxisInfo info, bool propogateToPlotRange)
+void AMScanViewSingleSpectrumView::onCheckBoxChanged(int id)
```

Comment:
I feel like the name of this function could be more descriptive. What do the check boxes represent?